tags/key.go

```diff
@@ -15,49 +15,112 @@
 
 package tags
 
-import "fmt"
-
-var keys []Key
+// Mutator modifies a tag set.
+type Mutator interface {
+	Mutate(t *TagSet) *TagSet
+}
 
 // Key represents a tag key.
```

Comment:
Worth documenting that a Key supports `==`.

Reply:
I haven't seen this documented before, take a look at the new godoc. Does that look good?